index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <header class="toolbar">
+            <div class="my-logo primary-color hover-underline"><a href="index.html">My Logo</a></div>
+            <nav class="collapsed_menu primary-color">&#9776;</nav>
+        </header>
+        <!--END OF TOOLBAR SECTION-->
+
+        <!--HEADLINE SECTION-->
+        <section>
+            <div>
+                <div class="ellipse_11"></div>
+                <div class="ellipse_5"> </div>
+                <div class="rectangle_7"> </div>
+                <div class="rectangle_4"> </div>
+                <div class="ellipse_3"> </div>
+            </div>
+            <div class="greetings">
+                <h1>I'm Naan.
```

Comment:
- [  ] Please as was requested by the last reviewer, fix these things on the headline section:- 
- the hamburger button doesn't appear on the design due to the background that you are using.
- instead of cutting the background into pieces you can just get the whole background as one piece and set its position.
![image](https://user-images.githubusercontent.com/74744787/219813273-14f24357-d613-413a-98ef-9a2a0d33b7c7.png)

- the distance between the social media icons and the text doesn't match the design.
![image](https://user-images.githubusercontent.com/74744787/219813778-47bc46e9-f4d6-43a1-a973-1db95dfc3437.png)

Reply:
![image](https://user-images.githubusercontent.com/124832435/219815310-8c8cc8ff-8d05-49f8-ab23-9d83c20f079d.png)
Hello @MeqdamAlqudah,
Hope you are having a great week.
I did not understand what you meant by the hamburger button.
- The previous reviewers did not give me a suggestion about the background so I assumed it was correct.
- I did fix the issues in the index.html file. The previous reviewer approved it too.
- The distance between the social media icons and the text matches the design on my computer(please the screenshot attached).
- Please check my project again. I have no more reviews left.